maas-api.yaml

```diff
@@ -1,555 +1,677 @@
-          default: (1, 1, 1, 1)
-        - name: ir
-          type: boolean
-          in: query
-          description: Boolean if interregional results, i.e., between regions as defined as in `regions.json`, should be returned.
-          default: false
-        - name: conc
-          description: If concession pricing should be use for public transport
-          type: boolean
-          in: query
-          default: false
-        - name: wheelchair
-          description: Boolean if routing results should include wheelchair information (note, that this will change 'walking' instruction to 'rolling' instruction).
-          type: boolean
-          in: query
-          default: false
-        - name: includeStops
-          type: boolean
-          in: query
-          description: Boolean if segment templates for public transport should include the list of all stops along the way, including *relative* times (i.e., seconds since start of the segment)
-          default: false
-        - name: 2wirc
-          type: boolean
-          in: query
-          description: Boolean to determine whether two-way-hire vehicles, such as pod-based car-share, should automatically add the cost of returning the vehicle to its pick-up location. Set to false if the cost of the trip should only include the cost that's attributed to this trip and ignore the unavoidable additional cost for returning the vehicle to its pick-up location.
-          default: true
-        - name: fb
-          type: boolean
-          in: query
-          description: _f_olding _b_icycle, boolean to determine whether the bicycle mode should be treated as a folding/portable bicycle. When this is set to `true`, the bicycle will be taken on any public transport mode, and will be taken all the way to the destination, rather than locking it up along the way. (Only applies if `modes` includes `cy_bic`.)
-          default: false
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/RoutingInput'
+
       responses:
         200:
           description: Successful response. Can include many trips.
-          schema:
-            $ref: '#/definitions/RoutingResponse'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/RoutingResponse'
+        400:
+          description: Bad request, most likely due to issues with the input
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/InputError'
+
+  /routing/grouped:
```

Comment:
can we merge this with `/routing` and add `group` as a parameter?

Reply:
No, different outputs schemas. It's discouraged for the same API endpoint for the same response type and the same HTTP status code to return different output schemas depending on the input.

Reply:
got it.  can we not use `/` as a separator then?  maybe `/trips` and `/trip-groups`?

Reply:
`/services/realtime` isn't working for me either.  maybe `/realtime/services` and `/realtime/alerts`?

Reply:
Done

Reply:
Missed the other comment: Why not use `/` as a separator?

Reply:
Just doesn't feel like a hierarchy.  /trips (/routing) and /trip-groups (/routing/grouped) seem like they should be at the same level.